app/src/main/java/net/gsantner/markor/util/ShortcutUtils.java

```diff
@@ -0,0 +1,131 @@
+/*#######################################################
+ *
+ *   Maintained by Gregor Santner, 2017-
+ *   https://gsantner.net/
+ *
+ *   License of this file: Apache 2.0 (Commercial upon request)
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+#########################################################*/
+package net.gsantner.markor.util;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.pm.ShortcutInfo;
+import android.content.pm.ShortcutManager;
+import android.graphics.drawable.Icon;
+import android.os.Build;
+import android.support.annotation.NonNull;
+
+import net.gsantner.markor.R;
+import net.gsantner.markor.activity.DocumentActivity;
+import net.gsantner.markor.activity.openeditor.OpenEditorQuickNoteActivity;
+import net.gsantner.markor.activity.openeditor.OpenEditorTodoActivity;
+import net.gsantner.markor.format.markdown.MarkdownTextConverter;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Utilities for app shortcuts.
+ * https://developer.android.com/guide/topics/ui/shortcuts
+ */
+public class ShortcutUtils {
+
+    private static final String ID_PREFIX = "shortcut_";
+    private static final String ID_TO_DO = ID_PREFIX + "to_do";
+    private static final String ID_QUICK_NOTE = ID_PREFIX + "quick_note";
+
+    private static final int MAX_RECENT_DOCUMENTS = 2;
+
+    private ShortcutUtils() {
+
+    }
+
+    /**
+     * Update the app shortcuts.
+     * The list will contain a link to to-do, QuickNote and 2 recent documents.
+     *
+     * @param context Context
+     */
+    public static void setShortcuts(@NonNull Context context) {
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.N_MR1) return;
+
+        ShortcutManager shortcutManager = context.getSystemService(ShortcutManager.class);
+        List<ShortcutInfo> newShortcuts = new ArrayList<>();
+
+        // Create the to-do shortcut
+        Intent intentToDo = new Intent(context, OpenEditorTodoActivity.class);
+        intentToDo.setAction(Intent.ACTION_VIEW);
+
+        ShortcutInfo shortcutToDo = new ShortcutInfo.Builder(context, ID_TO_DO)
```

Comment:
Know, the recommendation is 4, but the API supports 5 options.
Can you please add/try LinkBox to the static options?

Reply:
Just tried it. It's too much :(
`Caused by: java.lang.IllegalArgumentException: Max number of dynamic shortcuts exceeded`

Reply:
haha, ok thanks for trying :D.

Reply:
Also tried defining the static 3 shortcuts in XML and then adding 2 dynamic shortcuts. But then the static shortcuts get overwritten.
So now I've added LinkBox and removed one recent file.